service/src/event/service.rs

```diff
@@ -130,76 +177,35 @@ impl CeramicEventService {
         )?)
     }
 
-    async fn validate_signed_events(
-        events: Vec<EventInsertable>,
-    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
-        // TODO: IMPLEMENT THIS
-        Ok((events, Vec::new()))
-    }
-
-    async fn validate_time_events(
-        events: Vec<EventInsertable>,
-    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
-        // TODO: IMPLEMENT THIS
-        Ok((events, Vec::new()))
-    }
-
-    pub(crate) async fn validate_events(
```

Comment:
any reason to remove this helper function?  `insert_events` is quite long, having it broken down into a series of helper functions I think improves readability.

Reply:
I basically just moved it to the other file. It's currently not a static method and is `let validator = EventValidator::new(..); validator.validate_events(..).await;`. I'm going to rework things today and see how it shakes out - I didn't want to have this function that just called another function immediately and thought having a dedicated file might be nice since this one is getting long.

Maybe it makes sense to add a validator trait for this crate and inject the validator from recon/api/migration that composes the pieces rather than trying to have a smart validator that knows all of the nuances based on history requirements etc.